src/Symfony/Component/Form/Extension/Core/DataTransformer/NumberToLocalizedStringTransformer.php

```diff
@@ -93,13 +93,14 @@ class NumberToLocalizedStringTransformer implements DataTransformerInterface
      */
     const ROUND_HALFDOWN = \NumberFormatter::ROUND_HALFDOWN;
 
+    // todo rename to scale in Symfony 3.0
```

Comment:
But we could also make all props private in 3.0 as @stof suggested. Would be even better